agent/agent-bootstrap/src/main/java/com/microsoft/applicationinsights/agent/bootstrap/BytecodeUtil.java

```diff
@@ -111,25 +120,54 @@ public static long getTotalMilliseconds(long days, int hours, int minutes, int s
                 + milliseconds;
     }
 
+    // basically the same as SDK MapUtil.copy()
```

Comment:
we have duplicated code because of package accessibility limitation.  should we have a common framework like util framework to be shared across all modules? it can be used as a dependency.  this kind of refactoring can be done later.